packages/open-payments/src/client/ilp-stream-connection.test.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
not sure how important these tests in particular are, but I thought it was worth making sure we are adding the correct path for the response validator